bokehjs/src/coffee/models/plots/gmap_plot_canvas.coffee

```diff
@@ -110,8 +110,13 @@ export class GMapPlotCanvasView extends PlotCanvasView
     # create the map with above options in div
     @map = new maps.Map(@canvas_view.map_el, map_options)
 
-    # update bokeh ranges whenever the map idles, which should be after any UI action
-    maps.event.addListenerOnce(@map, 'idle', @_set_bokeh_ranges)
+    # update bokeh ranges whenever the map idles, which should be after most UI action
+    maps.event.addListener(@map, 'idle',
+      @_set_bokeh_ranges
+      @render())
```

Comment:
It does seem to perform better with a synchronous render but I can try again with a request to if it's acceptable when I push a change to make the callback syntax better. I'll submit a follow on PR in a short while.